apps/designer/app/designer/features/style-panel/shared/css-value-input/keyword-utils.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Beautify a CSS keyword by capitalizing the first letter of each word,
+ * and replacing dashes with spaces.
+ */
+export const toPascalCaseNoDashes = (keyword: string) => {
+  const label = keyword
+    .replace(/-/g, " ")
+    .split(" ")
+    .map((word) => {
+      return word[0].toUpperCase() + word.slice(1);
+    })
+    .join(" ");
+
+  return label;
+};
+
+/**
+ * Try to transform something from Pascal Case / Pascal Case / camelCase / camel Case to kebab-case
+ */
```

Comment:
In the future we can make search over all keywords we know, would be better as things like `mincontent` `alignleft` would work. (Now not)